gpt4all-chat/src/chatllm.cpp

```diff
@@ -738,6 +736,9 @@ bool ChatLLM::promptInternal(const QList<QString> &collectionList, const QString
     int32_t n_predict, int32_t top_k, float top_p, float min_p, float temp, int32_t n_batch, float repeat_penalty,
     int32_t repeat_penalty_tokens)
 {
+    if (!m_processedSystemPrompt)
+        processSystemPrompt();
+
     if (!isModelLoaded())
         return false;
```

Comment:
I know the effect is the same either way because processSystemPrompt() checks it too, but to me it would be more intuitive if the isModelLoaded() check came first.